src/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+p {
+  margin: 0;
+}
+
+body {
+  background: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+
+  --main-color: #060b35;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  color: var(--main-color);
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.stars-rewiew {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 .stars_star:nth-child(-n+1),
+.stars--2 .stars_star:nth-child(-n+2),
+.stars--3 .stars_star:nth-child(-n+3),
+.stars--4 .stars_star:nth-child(-n+4),
+.stars--5 .stars_star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.rewiew-text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.product-card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.product-card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.product-card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.product-card__buy-button {
+  border: 1px solid #00acdc;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 5px;
+  background: #00acdc;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  width: 100%;
+  height: 100%;
+}
+
+.product-card__buy-button:hover {
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  background: #fff;
+  text-transform: uppercase;
+  color: #00acdc;
```

Comment:
don't duplicate code. Write here only styles that are different from .product-card__buy-button class